js/ui/button_group.d.ts

```diff
@@ -174,6 +174,12 @@ export interface dxButtonGroupItem extends CollectionWidgetItem {
      * @public
      */
     type?: 'back' | 'danger' | 'default' | 'normal' | 'success';
+
+    /**
+     * @docid
+     * @public
+     */
+    elementAttr?: { [key: string]: any };
```

Comment:
This description inherits from dom component. Does we need it?

Reply:
No, collectionwidgetitem doesn't have this property. So, I think this property will not be inherit from DomComponent

![image](https://user-images.githubusercontent.com/57402891/129848092-9ec5ba3b-dde4-43b0-ab9c-1d05950a1cae.png)